### ISSUE TITLE
Text Alignment centre centre not working

### ISSUE THREAD

**Issue by weyert** [chunyenHuang/hummusRecipe#42]
I am trying to render a text in a text box but the vertically centred. Only I have been trying the following for a text only it keeps left aligned:

```
         {
             "x": 158,
             "y":  80,
             "style": {
                 "color": "592982",
                 "fontSize": 24,
                 "bold": true,
                 "font": "Helvetica",
                 "textBox": {
                    "textAlign": "left center",
                    "minHeight": 30,
                     "width": 400,
                    "lineHeight": 24,
                     "padding": [0, 0]
                 }
             }
         }
```

What am I doing wrong?

**Comment by chunyenHuang** [comment]
Sorry that I am pretty busy, I will take a look soon.

**Comment by weyert** [comment]
I have had a look but when I use something like:

```
                    "minHeight": 30,
                    "width": 275,
                    "height": 100,
                    "lineHeight": 12,
                    "padding": [0, 0],
                    "textAlign": "left bottom",
                    "style": {
                        "lineWidth": 1,
                        "stroke": "#00ff00"
                    }
```

I would expect that when the box is bigger then the actual rendered text I would expect the text to be docked to the bottom of the text box but even when I choose `right bottom` I don't notice any difference it's always rendered from the top left corner.

**Comment by Anandks1993** [comment]
I am having the same issue, need to center text both vertically and horizontally. Horizontal center is working fine, but not the vertical one. Is this issue solved ?

**Comment by shaehn** [comment]
Fixed in release 1.8.23
